mobile/native/src/dlc_handler.rs

```diff
@@ -115,22 +120,20 @@ impl DlcHandler {
                     temporary_channel_id,
                     ..
                 }) => {
-                    tracing::info!("Accepting pending dlc channel offer.");
+                    tracing::info!("Rejecting pending dlc channel offer.");
                     // Pending dlc channel offer not yet confirmed on-chain
 
                     event::publish(&EventInternal::BackgroundNotification(
                         BackgroundTask::RecoverDlc(TaskStatus::Pending),
                     ));
 
-                    if let Err(e) = self.node.accept_dlc_channel_offer(temporary_channel_id) {
-                        tracing::error!("Failed to accept pending dlc channel offer. {e:#}");
-                        tracing::warn!("Rejecting pending dlc channel offer!");
-                        self.node.reject_dlc_channel_offer(temporary_channel_id)?;
+                    self.node
+                        .reject_dlc_channel_offer(temporary_channel_id)
+                        .context("Failed to process pending dlc channel offer")?;
```

Comment:
🔧 So we've changed our policy here. Now we are rejecting all these pending offers when we reconnect. It would be good to reflect this in the commit message. Or even better, we could change the policies in a separate commit.

I thought this change was just about being _able_ to send these messages when necessary (and them having the expected effect on both sides). Not about changing policies.

What is the thinking behind this change of policy. I'm probably just forgetting an old conversation.

Reply:
I guess we should test this on-reconnect feature with e2e tests. Can you create a ticket for it?

Reply:
https://github.com/get10101/10101/issues/1940

Reply:
> Or even better, we could change the policies in a separate commit.

I split it up into separate commit messages and added a separate CHANGELOG entry.